containerd-shim-spin-v1/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "containerd-shim-spin-v1"
-version = "0.8.0"
+version = "0.9.0"
```

Comment:
Unforunately I don't thnk we current allow bumping invidiual shim's verison our release pipeline is using one version for all the shims.